ngraph/python/src/ngraph/impl/__init__.py

```diff
@@ -27,10 +27,15 @@
     # ngraph.dll is installed 3 directories above by default
     # and this path needs to be visible to the _pyngraph module
     #
-    # If you're using a custom installation of nGraph,
-    # add the location of ngraph.dll to your system PATH.
     ngraph_dll = os.path.join(os.path.dirname(__file__), "..", "..", "..")
-    os.environ["PATH"] = os.path.abspath(ngraph_dll) + ";" + os.environ["PATH"]
+
+    # PIP installs openvino dlls 2 directories above in openvino.libs by default
+    # and this path needs to be visible to the openvino modules
+    #
+    openvino_dll = os.path.join(os.path.dirname(__file__), "..", "..", "openvino.libs")
```

Comment:
If I remember right, nGraph doesn't depend on IE libraries. Why do you add such dependency?

Reply:
yes, ngraph doesn't depend on ie, but ie does. That is why we need to install all libs in one directory openvino.libs (ie and ngraph) for PIP.
But in this case _pyngraph.pyd will not be able to find ngraph.dll. And this change solves that by adding to the PATH variable alternative dll path.

Reply:
Why do we configure paths for openvino_dll inside the cmake for nGraph API?

Reply:
renamed and corrected comments